src/app/board/my-quests/my-quests.component.html

```diff
@@ -0,0 +1,15 @@
+<h2 *ngIf="acceptedQuests.length">
+  Hey, Hero! Start your
+  <span>{{ acceptedQuests.length > 1 ? "quests" : "quest" }}</span> today!
```

Comment:
Probably it would be helpful: you don't need `span` to use interpolation.

Reply:
hm.. probably I wanted to add some styles to this span, but forgot about it or was lazy =)